### ISSUE TITLE
Support typescript

### ISSUE THREAD

**Issue by lednhatkhanh** [tleunen/eslint-import-resolver-babel-module#99]
How can I config it to support typescript? The docs doesn't mention anything about this. Thanks.

**Comment by dawidk92** [comment]
Any updates about it?

**Comment by GeKorm** [comment]
Have you added the typescript extensions in your babel config? Here's my config for example, which works fine with @babel/preset-typescript

##### babel
```javascript
...
[   
  'babel-plugin-module-resolver',
  {
    cwd: 'packagejson',
    root: ['./src'],
    alias: {
      lib: './src/lib'
    },
    extensions: ['.js', '.ts']
  }
]
```

##### eslint
```json
"settings": {
  "import/resolver": {
    "node": {
      "paths": ["./src"],
    },
    "babel-module": {}
  }
}
```

**Comment by danielo515** [comment]
I did the other way around, by adding the extensions to te eslintrc config.
I think that the example on the README needs to be improved to include some bits of configuration. Right now it only has an empty object and that is kinda useless.
Also linking to [this documentation](https://github.com/tleunen/babel-plugin-module-resolver/blob/master/DOCS.md)  will probably be better because it is too hard on the linked one to find useful bits of config.

**Comment by Robin-Hoodie** [comment]
Bumped into this as well, the secret ingredient is adding the `extensions` option as mentioned above.

Babel itself doesn't complain when leaving out `extensions` but this plugin for some reason does.

The most elegant option is in my opinion to thus add this to the ESLint configuration

```yml
// .eslintrc.yml
settings:
  import/resolver:
    babel-module:
      extensions:
        - ".js"
        - ".ts"
        // Add other extensions if needed
```
